packaging/deb/init.d/grafana-server

```diff
@@ -33,6 +33,8 @@ DATA_DIR=/var/lib/grafana
 PLUGINS_DIR=/var/lib/grafana/plugins
 LOG_DIR=/var/log/grafana
 CONF_FILE=$CONF_DIR/grafana.ini
+DATASOURCES_CFG_DIR=$CONF_DIR/datasources
+DASHBOARDS_CFG_DIR=$CONF_DIR/dashboards
```

Comment:
I wonder if we it's wise to have separate configs for this. Maybe we will add users and panels as well in the future and that will be a lot of path configs. Maybe we can combine these to a provisioning dir? and under that dir grafana will create (if not exists) a datasources and dashboards dirs. alternatively we add a config_dir option that works the same but just more general than less specific than conf_file

Reply:
At first, I was hesitant. But I like this idea. Esp once we get beyond dahboards/datasources

How about:

[path]
~~dashboards~~ = 
~~datasources~~
provisioning = conf/provisioning


```bash
/etc/grafana/provisioning/dashboards
/etc/grafana/provisioning/datasources
/etc/grafana/provisioning/plugins
/etc/grafana/provisioning/alerts
/etc/grafana/provisioning/alert-notifiers
/etc/grafana/provisioning/template-vars
```